dev-util/rebar/rebar-3.22.1-r1.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN=${PN}3
+MECK_PV=0.8.13 # see rebar.config
+
+inherit bash-completion-r1
+
+DESCRIPTION="A sophisticated build-tool for Erlang projects that follows OTP principles"
+HOMEPAGE="https://www.rebar3.org https://github.com/erlang/rebar3"
+SRC_URI="
+	https://github.com/erlang/${MY_PN}/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz
+	test? ( https://repo.hex.pm/tarballs/meck-${MECK_PV}.tar )
+"
+S="${WORKDIR}"/${MY_PN}-${PV}
+
+LICENSE="Apache-2.0 MIT BSD"
+SLOT="3"
+KEYWORDS="amd64 ~arm ~ia64 ppc ppc64 sparc x86"
+IUSE="test"
+
+RESTRICT="!test? ( test )"
+
+# Note: /usr/bin/rebar is a ZIP archive of BEAM files so := is needed
+# see #913601
+RDEPEND="
+	dev-lang/erlang:=[ssl]
+	!dev-util/rebar-bin
```

Comment:
I'm wondering how long blockers should last if the package was already removed? `rebar-bin` was removed about ~1,5 years ago.. - maybe the blocker could be removed too. However maybe not in this PR, but with the next update..